python/GafferSceneUI/LightEditor.py

```diff
@@ -436,6 +578,28 @@ def __buttonPress( self, pathListing, event ) :
 					"command" : Gaffer.WeakMethod( self.__showHistory )
 				}
 			)
+			menuDefinition.append(
+				"Edit...",
+				{
+					"command" : functools.partial( self.__editSelectedCells, pathListing, False )
+				}
+			)
+			menuDefinition.append(
+				"Disable Edit",
+				{
+					"command" : functools.partial( self.__disableEdits, pathListing ),
+					"active" : lambda : len( self.__activeInspectionTweaks( pathListing ) ) == sum( [ p.size() for p in pathListing.getSelection() ] ),
+					"shortCut" : "D",
+				}
+			)
+			menuDefinition.append(
+				"Remove Attribute",
+				{
+					"command" : functools.partial( self.__removeAttributes, pathListing ),
+					"active" : lambda : len( self.__removableAttributeInspections( pathListing ) ) == sum( [ p.size() for p in pathListing.getSelection() ] ),
+					"shortCut" : "Delete",
```

Comment:
Looking at similar menu items, I think this needs to be `"Backspace, Delete"`, and `__keyPress` needs to be updated to match that.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/27a4928eaa58773ca5832d4ae05aac2dd0d4a666